### PR TITLE
LVM-290: Adds a navigation bar

### PR DESCRIPTION
**Pull request by nabeel-**
https://cs5500-jira.ccs.neu.edu/browse/LVM-290

### CODE REVIEW

File: app/views/layouts/_navigation.html.erb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<nav class="navbar navbar-default">
+  <div class="navbar-header">
+    <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
+    <span class="sr-only">Toggle navigation</span>
+    <span class="icon-bar"></span>
+    <span class="icon-bar"></span>
+    <span class="icon-bar"></span>
```
Comment (zack) [inline app/views/layouts/_navigation.html.erb:7]: what are these spans doing?

File: app/views/layouts/_navigation.html.erb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      <span>Literacy Volunteers of MA</span>
+    <% end %>
+  </div>
+  <div class="collapse navbar-collapse navbar-right">
+    <ul class="nav navbar-nav">
+      <% if current_user %>
+        <%= render 'layouts/navigation_links' %>
+      <% else %>
+        <li class="dropdown"><%= link_to "Login", new_user_session_path %></li>
+        <li class="dropdown"><%= link_to "Sign Up", new_user_registration_path %></li>
```
Comment (zack) [inline app/views/layouts/_navigation.html.erb]: Might as well remove this; Roberta said she was opting for the "admins create all users" method of user administration.

**Review by zack — APPROVED** [review]
Nonblocking comments. Nice!

File: app/views/layouts/_navigation.html.erb (hunk 1 above)
Comment (nabeel-) [reply app/views/layouts/_navigation.html.erb:7]: The little hamburger-type thing that people associate with opening menus.

screen shot 2017-01-30 at 3 45 32 pm

File: app/views/layouts/_navigation.html.erb (hunk 2 above)
Comment (nabeel-) [reply app/views/layouts/_navigation.html.erb]: Removed. Less confusion

**Pull request merged**